runner/run_test.go

```diff
@@ -10,6 +10,58 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestFakeGenericWorker(t *testing.T) {
+	defer filet.CleanUp(t)
+	dir := filet.TmpDir(t, "")
+	configPath := filepath.Join(dir, "runner.yaml")
+
+	exe := "go"
+	fakeWorker := "../hack/cp.go"
+	infile := filepath.Join(dir, "infile")
+	outfile := filepath.Join(dir, "outfile")
+	testData := "this is a test"
+	err := ioutil.WriteFile(infile, []byte(testData), 0755)
+	if !assert.NoError(t, err) {
```

Comment:
Have you heard about the [github.com/stretchr/testify/require](https://godoc.org/github.com/stretchr/testify/require) initiative?